src/library/styles/components/_button.scss

```diff
@@ -47,41 +47,3 @@
     }
   }
 }
-
-// Link buttons
-// This is not recommended -- we should use button styles on button elements, and link styles on link elements
-// NOTE: these will need to be updated if we modify the link mixins
-
-a {
```

Comment:
I think we should revert this since it only applies to link elements with specific classes, which we're using in library components like modal